src/Crosscompile.targets

```diff
@@ -31,6 +43,32 @@
 
   </Target>
 
+  <Target Name="SetPathToZig"
+          BeforeTargets="SetupOSSpecificProps">
+
+    <PropertyGroup Condition="$(HostRuntimeIdentifier.StartsWith('win-'))">
+      <ZigPath Condition="$(HostRuntimeIdentifier.EndsWith('-x86'))">$(PkgVezel_Zig_Toolsets_win-x86)/tools</ZigPath>
+      <ZigPath Condition="$(HostRuntimeIdentifier.EndsWith('-x64'))">$(PkgVezel_Zig_Toolsets_win-x64)/tools</ZigPath>
+      <ZigPath Condition="$(HostRuntimeIdentifier.EndsWith('-arm64'))">$(PkgVezel_Zig_Toolsets_win-arm64)/tools</ZigPath>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="$(HostRuntimeIdentifier.StartsWith('linux-'))">
+      <ZigPath Condition="$(HostRuntimeIdentifier.EndsWith('-x64'))">$(PkgVezel_Zig_Toolsets_linux-x64)/tools</ZigPath>
+      <ZigPath Condition="$(HostRuntimeIdentifier.EndsWith('-arm'))">$(PkgVezel_Zig_Toolsets_linux-arm)/tools</ZigPath>
+      <ZigPath Condition="$(HostRuntimeIdentifier.EndsWith('-arm64'))">$(PkgVezel_Zig_Toolsets_linux-arm64)/tools</ZigPath>
+      <ZigPath Condition="$(HostRuntimeIdentifier.EndsWith('-ppc64le'))">$(PkgVezel_Zig_Toolsets_linux-ppc64le)/tools</ZigPath>
+      <ZigPath Condition="$(HostRuntimeIdentifier.EndsWith('-riscv64'))">$(PkgVezel_Zig_Toolsets_linux-riscv64)/tools</ZigPath>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="$(HostRuntimeIdentifier.StartsWith('osx-'))">
+      <ZigPath Condition="$(HostRuntimeIdentifier.EndsWith('-x64'))">$(PkgVezel_Zig_Toolsets_osx-x64)/tools</ZigPath>
+      <ZigPath Condition="$(HostRuntimeIdentifier.EndsWith('-arm64'))">$(PkgVezel_Zig_Toolsets_osx-arm64)/tools</ZigPath>
+    </PropertyGroup>
```

Comment:
This is all dead code without making some kind of clang wrapper that redirects to zig (clang.cmd does this on Windows). I'd prefer dropping this part.